Jenkinsfile

```diff
@@ -8,7 +8,7 @@ stage("Sanity Check") {
       source activate ./conda/lint
       conda list
       make clean
-      make pylint && python setup.py check --restructuredtext --strict
```

Comment:
Why remove pylint?  Why not just keep it in and do both __make pylint__ and __make lint__?  That would form a nice "double check" because they are two different tools that find different sets of issues.

Reply:
`make lint` will run pylint, setup.py check and flake8. I have disabled all but the last now to see if the recursionerror still occurs.